src/Grpc.AspNetCore.Server/IGrpcInterceptorActivator.cs

```diff
@@ -16,6 +16,7 @@
 
 #endregion
 
+using System;
 using Grpc.Core.Interceptors;
 
 namespace Grpc.AspNetCore.Server
```

Comment:
I believe we wanted to make these activators internal. I see that it's being shared with the Reflection package though.

Reply:
TODO

Reply:
TO**DONE**